### PR TITLE
feat: add support for countryCode config option

### PR DESCRIPTION
**Pull request by yeul**

### CODE REVIEW

File: src/messages/services/banner/index.js [hunk 1]
```diff
@@ -57,12 +57,27 @@ function mutateMarkup(markup) {
     }
 }
 
-function fetcherB({ account, merchantId, amount, offerType, currency, style: { typeEZP } }) {
```
Comment (merlinstardust) [inline src/messages/services/banner/index.js:60]: So I can see that this function isn't new, but why is it call fetcherB? Can a comment be added here?

**Review by merlinstardust — APPROVED** [review]
This looks good, I'm just curious about the `fetcherB` name.

File: src/messages/services/banner/index.js (hunk 1 above)
Comment (Seavenly) [reply src/messages/services/banner/index.js:60]: `fetcherA` and `fetcherB` are labeled as such just fo the ramp. Eventually they will be condensed back into a single `fetcher`. They are not supposed to have a long lifespan in the code.

File: src/messages/models/Banner/validateOptions.js [hunk 2]
```diff
@@ -38,9 +39,18 @@ function getValidVal(logger, typeArr, val, location) {
...
+            const validVal = arrayFind(validVals, v => {
+                if (typeof v !== 'string') {
+                    return false;
+                }
+                return v.split('|').some(x => x === val);
+            });
+
             if (validVal === undefined) {
                 logInvalidOption(logger, location, validVals, val);
+                if (validVals[0] === undefined) {
```
Comment (Seavenly) [inline src/messages/models/Banner/validateOptions.js]: Consider `typeof validVals[0] === 'undefined'` instead when checking for `undefined`

**Pull request merged**